js/components/frame.js

```diff
@@ -140,29 +140,28 @@ class Frame extends ImmutableComponent {
     this.webview.addEventListener('new-window', (e, url, frameName, disposition, options) => {
       e.preventDefault()
 
-      const guestInstanceId = e.options && e.options.webPreferences && e.options.webPreferences.guestInstanceId
-      const windowOptions = e.options && e.options.windowOptions || {}
-      windowOptions.parentWindowKey = remote.getCurrentWindow().id
-      windowOptions.disposition = e.disposition
+      let guestInstanceId = e.options && e.options.webPreferences && e.options.webPreferences.guestInstanceId
+      let windowOpts = e.options && e.options.windowOptions || {}
+      windowOpts.parentWindowKey = remote.getCurrentWindow().id
+      windowOpts.disposition = e.disposition
+      let delayedLoadUrl = e.options && e.options.delayedLoadUrl
+
+      let frameOpts = {
+        location: e.url,
+        parentFrameKey: this.props.frame.get('key'),
+        isPrivate: this.props.frame.get('isPrivate'),
+        partitionNumber: this.props.frame.get('partitionNumber'),
+        // use the delayed load url for the temporary title
+        title: delayedLoadUrl,
```

Comment:
Since this i set to the title in the state I think it will cause title mode to display still.

Usually when a page has no title the UI should not go into title mode. 
I.e. showing

somedomain.com | http://somedomain.com/test 
would be wrong and it's preferred to just show the urlbar.